influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -342,6 +327,57 @@ impl<W: Wal, T: TimeProvider, P: Persister> WriteBufferImpl<W, T, P> {
         Ok(chunks)
     }
 
+    pub async fn cache_parquet(
+        &self,
+        db_name: &str,
+        table_name: &str,
+        records: SendableRecordBatchStream,
+    ) -> Result<(), Error> {
+        let (min_time, max_time) = self.segment_state.read().open_segment_min_max_times();
```

Comment:
I think this function should have these times as arguments. The caller will have the table specific min/max times for whatever is being cached, which might not be the entire segment buffer for the table.

Reply:
ahh okay that I wasn't sure about. That makes this a bit easier then.